crates/hyperdrive-math/src/long/targeted.rs

```diff
@@ -122,6 +125,8 @@ impl State {
                     "We overshot the zero-crossing during Newton's method.",
                 ));
             }
+            // The optimization loss can be the difference without abs or squaring
+            // because of the above check.
```

Comment:
This seems like comment overkill since the line it's explaining immediately follows the referenced check. It took me longer to make sense of what the comment was trying to clarify than just reading the code without it.